### PR TITLE
Support Alpine Linux & OpenRC

### LINKED ISSUE #50: Support alpine linux
Add installation support for Alpine linux.  This would be very helpful when docker containers need to be used in a CI environment.

### PR DESCRIPTION
**Pull request by tomalok**
*Issue #, if available:*
Resolves #50

*Description of changes:*
Adds support for Alpine Linux / OpenRC

### CODE REVIEW

**Comment by tomalok** [comment]
@Cappuccinuo i've been running with this patch on Alpine for some time now -- would be great if this got merged...  ;)

**Comment by Cappuccinuo** [comment]
Thanks @tomalok for the PR, we'll find someone to work on this. Can you provide the AMI you are using?

File: src/mount_efs/__init__.py [hunk 1]
```diff
@@ -1062,6 +1063,10 @@ def get_init_system(comm_file='/proc/1/comm'):
                 init_system = f.read().strip()
         except IOError:
             logging.warning('Unable to read %s', comm_file)
+
+        if init_system == 'init':
+            if os.path.isfile('/sbin/openrc'):
```
Comment (Cappuccinuo) [inline src/mount_efs/__init__.py:1068]: Can combine these into one line

```python
if init_system == 'init' and os.path.isfile('/sbin/openrc'):
   ...
```

Also can you add some logging here that indicate we hack the init_system to be `openrc-init`?

Additionally, can you add some comment on top that introduce that this is to support openrc or something like that, so we know why we are doing this.

**Review by Cappuccinuo — CHANGES_REQUESTED** [review]
LGTM in general. Minor comments.

File: src/mount_efs/__init__.py (hunk 1 above)
Comment (tomalok) [reply src/mount_efs/__init__.py:1068]: thanks for (finally!) getting at this!  :)

i will double-check the patch vs. the latest released Alpine 3.15 cloud images to make sure this patch is still correct on that end, and fix the above.

**Comment by tomalok** [comment]
> Thanks @tomalok for the PR, we'll find someone to work on this. Can you provide the AMI you are using?

Previously I was applying the patch as a customization on top of https://gitlab.alpinelinux.org/alpine/cloud/alpine-ec2-ami, which until recently was building the official Alpine Linux AMIs.

With Alpine Linux 3.15.0, the official bulid system for cloud images in general has changed (i have some docs to write before putting it at https://gitlab.alpinelinux.org/alpine/cloud/alpine-cloud-images -- hopefully will be moved there in a couple days) - this new build system also includes the potential for building customized images, I'll adapt these patches (with suggested fixes), and verify soonish.

File: src/mount_efs/__init__.py (hunk 1 above)
Comment (tomalok) [reply src/mount_efs/__init__.py]: Requested changes should be completed now.

**Comment by tomalok** [comment]
@Cappuccinuo & @JigarDedhia  - I've updated the PR to resolve conflicts.

**Comment by tomalok** [comment]
Obsoleted by #136 (and other merged changes that detect whether or not to use libwrap).

**Pull request closed without merging**